pkg/resource/deploy/import.go

```diff
@@ -73,8 +73,10 @@ func NewImportPlan(ctx *plugin.Context, target *Target, projectName tokens.Packa
 		return nil, err
 	}
 
+	builtins := newBuiltinProvider(nil, nil)
```

Comment:
Is a `nil` backendClient legal?  Are we sure that can't trigger panics in any cases `import` might hit?

Reply:
I'm not 100% certain about that, no. I'm exploring the downstream code now, but I wanted to get this start out ASAP.

Reply:
I've verified that all uses of the backend client are nil-checked, so this won't hit any panics. Working on tests now.